src/gromacs/nblib/tests/nbnxnsetup.cpp

```diff
@@ -48,17 +48,23 @@
 
 #include "testutils/testasserts.h"
 
+#include "testhelpers.h"
+#include "testsystems.h"
+
 namespace nblib
 {
 
 namespace test
 {
-/*
+
 TEST(NBlibTest, NbfpSetCorrectly)
```

Comment:
This should be CanConstructNbvSetupUtil and should test that construction does not throw.